### PR TITLE
[doc] Use "param" instead of "code"  to refer to parameters

### PR DESCRIPTION
**Pull request by asmaloney**
Also fixes a couple of descriptions in Vector4.xml and Viewport.xml.

This is the first of several commits to fix all of the class XML docs.

(Work based on discussion with @YuriSizov.)

### CODE REVIEW

File: doc/classes/UndoRedo.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
 			<param index="1" name="property" type="StringName" />
 			<param index="2" name="value" type="Variant" />
 			<description>
-				Register a property value change for "do".
+				Register a [param property] [param value] change for "do".
```
Comment (YuriSizov) [inline doc/classes/UndoRedo.xml:79]:
```suggestion
				Register a [param property] that would change its value to [param value] when the action is committed.
```

File: doc/classes/UndoRedo.xml [hunk 2]
```diff
@@ -100,7 +100,7 @@
 			<param index="1" name="property" type="StringName" />
 			<param index="2" name="value" type="Variant" />
 			<description>
-				Register a property value change for "undo".
+				Register a [param property] [param value] change for "undo".
```
Comment (YuriSizov) [inline doc/classes/UndoRedo.xml:103]:
```suggestion
				Register a [param property] that would change its value to [param value] when the action is undone.
```

File: doc/classes/UndoRedo.xml [hunk 3]
```diff
@@ -115,14 +115,14 @@
 			<param index="0" name="increase_version" type="bool" default="true" />
 			<description>
 				Clear the undo/redo history and associated references.
-				Passing [code]false[/code] to [code]increase_version[/code] will prevent the version number to be increased from this.
+				Passing [code]false[/code] to [param increase_version] will prevent the version number to be increased from this.
```
Comment (YuriSizov) [inline doc/classes/UndoRedo.xml:118]:
```suggestion
				Passing [code]false[/code] to [param increase_version] will prevent the version number from increasing when the history is cleared.
```

File: doc/classes/UndoRedo.xml [hunk 4]
```diff
@@ -115,14 +115,14 @@
...
 			</description>
 		</method>
 		<method name="commit_action">
 			<return type="void" />
 			<param index="0" name="execute" type="bool" default="true" />
 			<description>
-				Commit the action. If [code]execute[/code] is true (default), all "do" methods/properties are called/set when this function is called.
+				Commit the action. If [param execute] is true (default), all "do" methods/properties are called/set when this function is called.
```
Comment (YuriSizov) [inline doc/classes/UndoRedo.xml:125]:
```suggestion
				Commit the action. If [param execute] is [code]true[/code] (which it is by default), all "do" methods/properties are called/set when this function is called.
```

File: doc/classes/UndoRedo.xml [hunk 5]
```diff
@@ -131,7 +131,7 @@
 			<param index="1" name="merge_mode" type="int" enum="UndoRedo.MergeMode" default="0" />
 			<description>
 				Create a new action. After this is called, do all your calls to [method add_do_method], [method add_undo_method], [method add_do_property], and [method add_undo_property], then commit the action with [method commit_action].
-				The way actions are merged is dictated by the [code]merge_mode[/code] argument. See [enum MergeMode] for details.
+				The way actions are merged is dictated by the [param merge_mode] argument. See [enum MergeMode] for details.
```
Comment (YuriSizov) [inline doc/classes/UndoRedo.xml:134]:
```suggestion
				The way actions are merged is dictated by [param merge_mode]. See [enum MergeMode] for details.
```

File: doc/classes/XMLParser.xml [hunk 6]
```diff
@@ -19,14 +19,14 @@
 			<return type="String" />
 			<param index="0" name="idx" type="int" />
 			<description>
-				Gets the name of the attribute specified by the index in [code]idx[/code] argument.
+				Gets the name of the attribute specified by the index ([param idx]).
```
Comment (YuriSizov) [inline doc/classes/XMLParser.xml:22]:
```suggestion
				Gets the name of the attribute specified by the [param idx] index.
```

File: doc/classes/XMLParser.xml [hunk 7]
```diff
@@ -19,14 +19,14 @@
...
 			</description>
 		</method>
 		<method name="get_attribute_value" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="idx" type="int" />
 			<description>
-				Gets the value of the attribute specified by the index in [code]idx[/code] argument.
+				Gets the value of the attribute specified by the index ([param idx]).
```
Comment (YuriSizov) [inline doc/classes/XMLParser.xml:29]:
```suggestion
				Gets the value of the attribute specified by the [param idx] index.
```

File: doc/classes/XRPositionalTracker.xml [hunk 8]
```diff
@@ -23,14 +23,14 @@
...
-				Returns the current [XRPose] state object for the bound [code]pose[/code].
+				Returns the current [XRPose] state object for the bound [param name].
 			</description>
 		</method>
 		<method name="has_pose" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="name" type="StringName" />
 			<description>
-				Returns [code]true[/code] if the bound [code]tracker[/code] is available and is currently tracking the bound [code]pose[/code].
+				Returns [code]true[/code] if the bound [code]tracker[/code] is available and is currently tracking the bound [param name].
```
Comment (YuriSizov) [inline doc/classes/XRPositionalTracker.xml:33]:
```suggestion
				Returns [code]true[/code] if the tracker is available and is currently tracking the bound [param name] pose.
```

Seems to refer to itself, not sure why it's in the code tag.

File: doc/classes/XRPositionalTracker.xml [hunk 9]
```diff
@@ -23,14 +23,14 @@
 			<return type="XRPose" />
 			<param index="0" name="name" type="StringName" />
 			<description>
-				Returns the current [XRPose] state object for the bound [code]pose[/code].
+				Returns the current [XRPose] state object for the bound [param name].
```
Comment (YuriSizov) [inline doc/classes/XRPositionalTracker.xml:26]:
```suggestion
				Returns the current [XRPose] state object for the bound [param name] pose.
```

File: doc/classes/XRServer.xml [hunk 10]
```diff
@@ -54,7 +54,7 @@
 			<return type="XRInterface" />
 			<param index="0" name="idx" type="int" />
 			<description>
-				Returns the interface registered at a given index in our list of interfaces.
+				Returns the interface registered at a given index ([param idx]) in our list of interfaces.
```
Comment (YuriSizov) [inline doc/classes/XRServer.xml:57]:
```suggestion
				Returns the interface registered at the given [param idx] index in the list of interfaces.
```

**Review by YuriSizov — CHANGES_REQUESTED** [review]
Great work! Some additional nitpicks/language and formatting adjustments, but otherwise this is good!

**Review by YuriSizov — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**